Android/app/src/main/java/fashiome/android/models/Product.java

```diff
@@ -136,47 +112,87 @@ public String getCurrency() {
         return this.currency;
     }
 
-    public void setCurrency(String currency) {
-        this.currency = currency;
-        put("currency", currency);
+    public String getProductName() {
+
+        if (this.productName == null) {
+            this.productName = getString("productName");
+        }
+        String string= this.productName;
+        return string;
     }
 
-    public String getProductDescription() {
+    public String getProductSize() {
 
-        if (this.productDescription == null) {
-            this.productDescription = getString("productDescription");
+        if (this.productSize == null) {
+            this.productSize = getString("productSize");
         }
-        String string= this.productDescription;
+        String string = this.productSize;
         return string;
     }
 
+    public List<String> getPhotos() {
```

Comment:
you can get the photos from the cloudinary method. any specific need to add this ?